lib/rust/ensogl/component/list-view/src/entry/list.rs

```diff
@@ -158,13 +165,30 @@ where E::Model: Default
         }
     }
 
+    /// Update displayed entries to use styles located at the `style_prefix` path in the
+    /// application's style sheet.
```

Comment:
I would say explicitly here, that we _recreate_ them, to restrain any potential user from changing style too many times.